cql2pgjson/src/test/resources/compoundIndex.sql

```diff
@@ -27,6 +30,18 @@ CREATE INDEX IF NOT EXISTS tableb_idx_gin ON tableb using gin
 
 CREATE INDEX IF NOT EXISTS tablea_idx_ft ON tableb using gin
     ( to_tsvector('simple',concat_space_sql(tableb.jsonb->>'field1',tableb.jsonb->>'field2')));
+CREATE INDEX IF NOT EXISTS tablec_idx_gin ON tablec using gin
+    ((concat_space_sql(tablec.jsonb->>'firstName',tablec.jsonb->>'lastName')) gin_trgm_ops);
+    
+CREATE INDEX IF NOT EXISTS tablec_idx_ft ON tablec using gin
+    ( to_tsvector('simple',concat_space_sql(tablec.jsonb->>'field1',tablec.jsonb->>'field2')));
+    
+CREATE INDEX IF NOT EXISTS tabled_idx_gin ON tabled using gin
+    ((concat_space_sql(tabled.jsonb->>'city',tabled.jsonb->>'state')) gin_trgm_ops);
+
+CREATE INDEX IF NOT EXISTS tabled_idx_ft ON tabled using gin
+    ( to_tsvector('simple',concat_space_sql(tabled.jsonb->>'field1',tabled.jsonb->>'field2')));
+    
```

Comment:
Can you remove all the CREATE INDEX statements? They are not used in the unit tests.